languages/rust/oso/examples/lib_guide.rs

```diff
@@ -168,7 +168,7 @@ fn iters() -> anyhow::Result<()> {
     )
     .unwrap();
 
-    oso.load_str(r#"allow(actor, action, resource) if actor.get_group() = "payroll";"#)?;
```

Comment:
```suggestion
    oso.load_str(r#"allow(actor, action, resource) if actor.get_group() = "payroll";"#)?;
```

Giỗ tổ Hùng Vương, Thứ Tư, 21 thg 3

Reply:
@hombyk-nevzi2-dAxzyd